tangelo/toolboxes/molecular_computation/tests/test_fno.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2023 Good Chemistry Company.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy as np
+
+from tangelo import SecondQuantizedMolecule
+from tangelo.molecule_library import mol_H2_321g, mol_H2_sto3g_uhf
+from tangelo.toolboxes.molecular_computation.fno import FNO
+
+
+class FNOTest(unittest.TestCase):
+
+    def test_h2_321g_fno_restricted(self):
+        """Test the FNO active space selection and the molecular coefficients
+        for the H2/3-21G molecular system, with a restricted mean-field.
+        """
+
+        fno = FNO(mol_H2_321g, 0.50)
+        frozen_orbitals = fno.get_frozen_indices()
+
+        self.assertEqual([2, 3], frozen_orbitals)
+
+        fno_mo_coeff_ref = np.array([
```

Comment:
Any chance that this may be up to a sign / permutation in molecular calculations in the mean field or something ? Technically the molecule objects are computed when  the module molecular_library is imported

ignore this comment otherwise.